internal/rest/router.go

```diff
@@ -101,6 +102,7 @@ func (r *router) addRoutes() {
 
 	r.httpRouter.GET("/ws", r.wsHandler)
 	r.httpRouter.GET("/status", r.statusHandler)
+	r.httpRouter.POST("/pprof", r.dumpGoRoutines)
```

Comment:
Should this be enabled all the time? I'm thinking this isn't something people would want on in production and should be configurable. It also looks like it's implemented a little differently here than in other FireFly services

Reply:
good point. see the latest commit that moves it to the debug port and different routes